src/pages/index.js

```diff
@@ -30,53 +74,22 @@ const addImagePopup = new PopupWithImage("#view-image-modal");
 
 addImagePopup.setEventListeners();
 
-const userInfo = new UserInfo("#profile-name", "#profile-description");
+const ChangeProfilePicture = new PopupWithForm(
```

Comment:
`ChangeProfilePicture`

Every variable in the project should be in `camelCase` (in **lower case** if it's not a class or component)